chain/lf3/manifest.go

```diff
@@ -11,30 +11,23 @@ import (
 
 	"github.com/filecoin-project/lotus/build"
 	"github.com/filecoin-project/lotus/build/buildconstants"
-	"github.com/filecoin-project/lotus/chain/stmgr"
-	"github.com/filecoin-project/lotus/chain/store"
 	"github.com/filecoin-project/lotus/node/modules/dtypes"
 )
 
-func NewManifestProvider(nn dtypes.NetworkName, cs *store.ChainStore, sm *stmgr.StateManager, ps *pubsub.PubSub) manifest.ManifestProvider {
+func NewManifestProvider(nn dtypes.NetworkName, ps *pubsub.PubSub) manifest.ManifestProvider {
 	m := manifest.LocalDevnetManifest()
```

Comment:
@masih both `MinimumPollInterval` and `MaximumPollInterval` are based on `DefaultEcConfig.ECPeriod` in go-f3 but we don't reset them here. _Should_ we? I believe they come out the same here as long as `BlockDelaySecs` is `30`. How much do these matter to align to epoch length?

Reply:
>  I believe they come out the same here as long as `BlockDelaySecs` is 30

I think regardless of what `BlockDelaySecs` is set to cert exchange min max would be 30s and 2m respectively, because their value is instantiated via `DefaultCxConfig` var using `DefaultEcConfig.ECPeriod`.

For example, this passes for me.

```go
import (
	"testing"
	"time"

	"github.com/filecoin-project/go-f3/manifest"
)

func TestMinMaxPollIntervalIsNotOverridden(t *testing.T) {
	m := manifest.LocalDevnetManifest()
	m.ECPeriod = 1
	if m.CxConfig.MinimumPollInterval != 30*time.Second {
		t.Fatalf("min ain't it")
	}
	if m.CxConfig.MaximumPollInterval != 4*30*time.Second {
		t.Fatalf("max ain't it")
	}
}
```

> How much do these matter to align to epoch length?

Cert exchange internally optimises for the right poll time. As long as min and max are a wide enough interval of time, and min is not less than an epoch we should be good.

@Stebalien does that sound right?